doc/news/_preparation_next_release.md

```diff
@@ -88,6 +88,9 @@ We added even more functionality, which could not make it to the highlights:
 ## Other News
 
 - The `crypto` plugin now uses Elektra's `libinvoke` and the `base64` plugin in order to encode and decode Base64 strings. This improvement reduces code duplication between the two plugins. *(Peter Nirschl)*
+- The [`list` plugin](http://libelektra.org/plugins/list) now allows to pass
+  common configuration for plugins below the "config/" setting. The updated
```

Comment:
What about something like:

for plugins below the "config/" setting
->
for all plugins by using keys below the "config/" setting